README.md

```diff
@@ -2098,14 +2115,7 @@ set :protection, :except => [:path_traversal, :session_hijacking]
 ```
 
 By default, Sinatra will only set up session based protection if `:sessions`
-has been enabled. Sometimes you want to set up sessions on your own, though. In
-that case you can get it to set up session based protections by passing the
-`:session` option:
-
-```ruby
-use Rack::Session::Pool
-set :protection, :session => true
-```
+have been enabled. See 'Using Sessions'.
```

Comment:
Like I was saying before, if this example doesn't work I'd like to add a regression test if you're up for it!